processor/probabilisticsamplerprocessor/metadata.yaml

```diff
@@ -12,8 +12,20 @@ status:
 
 tests:
   config:
-  goleak:
-    ignore:
-      top:
-        # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
-        - "go.opencensus.io/stats/view.(*worker).start"
+
+telemetry:
+  metrics:
+    count_traces_sampled:
```

Comment:
I think this needs the otelcol_processor prefix. Can you confirm with the metrics we have today?

Reply:
Metrics before the change:

```
# HELP otelcol_processor_probabilistic_sampler_count_traces_sampled Count of traces that were sampled or not
# TYPE otelcol_processor_probabilistic_sampler_count_traces_sampled counter
otelcol_processor_probabilistic_sampler_count_traces_sampled{policy="missing_randomness",sampled="false",service_instance_id="f1a27d2b-5019-44b7-8dbb-b35b6f852c8c",service_name="otelcontribcol",service_version="0.104.0-dev"} 2
otelcol_processor_probabilistic_sampler_count_traces_sampled{policy="trace_id_hash",sampled="true",service_instance_id="f1a27d2b-5019-44b7-8dbb-b35b6f852c8c",service_name="otelcontribcol",service_version="0.104.0-dev"} 2
```

After the change:

```
# HELP otelcol_processor_probabilistic_count_logs_sampled Count of logs that were sampled or not
# TYPE otelcol_processor_probabilistic_count_logs_sampled counter
otelcol_processor_probabilistic_count_logs_sampled{policy="missing_randomness",sampled="false",service_instance_id="2a33aa5b-3531-4651-a3fb-c641900d28cc",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
# HELP otelcol_processor_probabilistic_count_traces_sampled Count of traces that were sampled or not
# TYPE otelcol_processor_probabilistic_count_traces_sampled counter
otelcol_processor_probabilistic_count_traces_sampled{policy="trace_id_hash",sampled="true",service_instance_id="2a33aa5b-3531-4651-a3fb-c641900d28cc",service_name="otelcontribcol",service_version="0.104.0-dev"} 2
```

Reply:
Note the change in the name of the logs vs traces sampled counter is a bugfix